src/uri/IpAddress.cpp

```diff
@@ -56,14 +56,11 @@ void IpAddress::toBytes() {
  */
 void IpAddress::toString() {
     if (!ipBytes_.empty()) {
-        
-        ipString_.reserve(INET6_ADDRSTRLEN + 1);
-
-        auto inetType = (type_ == AddressType::IpV4) ? AF_INET : AF_INET6;
-
-        if (inet_ntop(inetType, &ipBytes_[0], ipString_.data(), INET6_ADDRSTRLEN) == nullptr) {
-            spdlog::error("inet_ntop failed");
-        }
+	    auto inetType = (type_ == AddressType::IpV4) ? AF_INET : AF_INET6;
+	    if (std::string ipString(INET6_ADDRSTRLEN, '\0');
```

Comment:
How does this line of code compile?

Reply:
Its not a complete line. Line#60 and #61 is a complete line. 
C++17 allows if statement initialization.